src/commands/config.ts

```diff
@@ -0,0 +1,73 @@
+import { Channel, CommandInteraction, Interaction } from "discord.js";
+import { Db } from "mongodb";
+import { MessageExecutable } from "./command";
+
+import botConfig from "../config/botConfig";
+
+class DefaultChannelSubcommand extends MessageExecutable<CommandInteraction> {
+  async execute() {
+    const interaction_user = this.interaction.user;
+    const role = this.interaction.guild.roles.cache.find(
```

Comment:
At some point I think we definitely wanna get this in some sort of `AdminAction` object to abstract this away. Up to you if you want to figure this out in this PR or in a future contrib

Reply:
I'm thinking leave it for a future issue, I'd rather get the feature in now since it's more of a nice-to-have on the development side, while this is a deliverable.